### ISSUE TITLE
Different varargs typing in typescript

### ISSUE THREAD

**Issue by dbo** [eclipse/n4js#1277]
TS: `function foo(...bar: string[])`
N4JS: `function foo(...bar: string)`

**Comment by jpilgrim** [comment]
@dbo This is a break change... Do you have any idea how we could provide a nice update mechanism?

**Comment by dbo** [comment]
No, not really. Maybe the usual quickfix that IDE usually offered for a limited period of time.
